# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1308526247 25200
#      Sun Jun 19 16:30:47 2011 -0700
# Branch stable
# Node ID 59f5c239b9cf63e7b7bc4d263b029287bbe8bbda
# Parent  5ee4fc58585fe11b0e8c351cb1b8f4436968b43b
src/Makefile.am: Remove Makefile rules already implied by Automake usage.

* src/Makefile.am: Remove bin_SCRIPTS, bin_PROGRAMS from all-local rule,
remove bin_PROGRAMS from CLEANFILES variable.

diff --git a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -813,17 +813,17 @@ doc-files: $(ALL_DEF_FILES)
 gendoc.cc: doc-files mkgendoc
 	$(srcdir)/mkgendoc doc-files > $@-t
 	mv $@-t $@
 
 gendoc$(BUILD_EXEEXT): gendoc.cc
 	$(BUILD_CXX) $(BUILD_CXXFLAGS) -o $@ $^ $(BUILD_LDFLAGS)
 
 
-all-local: $(bin_PROGRAMS) $(bin_SCRIPTS) $(OCT_STAMP_FILES) DLD-FUNCTIONS/PKG_ADD .DOCSTRINGS
+all-local: $(OCT_STAMP_FILES) DLD-FUNCTIONS/PKG_ADD .DOCSTRINGS
 
 if AMCOND_BUILD_COMPILED_AUX_PROGRAMS
 octave-config.cc: octave-config.cc.in Makefile
 	@$(do_subst_default_vals)
 
 mkoctfile.cc: mkoctfile.cc.in Makefile
 	@$(do_subst_config_vals)
 else
@@ -894,17 +894,16 @@ uninstall-oct:
 	for f in $(notdir $(OCT_FILES)); do \
 	  rm -f $(DESTDIR)$(octfiledir)/$$f; \
 	done
 	rm -f $(DESTDIR)$(octfiledir)/PKG_ADD
 endif
 .PHONY: install-oct uninstall-oct
 
 CLEANFILES = \
-  $(bin_PROGRAMS) \
   $(bin_SCRIPTS) \
   DLD-FUNCTIONS/PKG_ADD \
   doc-files \
   gendoc.cc \
   gendoc$(BUILD_EXEEXT) \
   graphics-props.cc \
   oct-parse.output
 
